libraries/stdlib/jvm/test/random/RandomJvmSerializationTest.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package random
+
+import test.io.serializeAndDeserialize
+import kotlin.random.Random
+import kotlin.random.asJavaRandom
+import kotlin.random.asKotlinRandom
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertSame
+
+class RandomJvmSerializationTest {
```

Comment:
After we settle with serialized representation and serialVersionUID, let's add some serialization stability tests.
Something like https://github.com/JetBrains/kotlin/blob/1.4.20/libraries/stdlib/jvm/test/collections/CollectionJVMTest.kt#L192-L207